interface/docker-based-rock-ons/scrutiny.rst

```diff
@@ -195,14 +194,14 @@ installation.
    :align: center
 
 
-Step 5, installation in progress, you can close the wizzard.
+Step 5, with the installation in progress, close the wizard.
 
 .. image:: /images/interface/docker-based-rock-ons/scrutiny_install_step5.png
    :width: 100%
    :align: center
 
 
-Once the installation done, you can use the **Scrutiny UI** button to go to the
+Once the installation done, use the **Scrutiny UI** button to go to the
```

Comment:
```suggestion
Once the installation finishes, use the **Scrutiny UI** button to go to the
```

Reply:
Minor grammatical suggestion.

Reply:
Makes perfect sense. I think I did not touch that portion from the original commit, but, yes.

Reply:
Yes, my apologies for scope creep on this one.